src/work-allocation-2/containers/task-assignment-confirm/task-assignment-confirm.component.html

```diff
@@ -1,13 +1,13 @@
 <main class="govuk-main-wrapper" id="main-content" role="main">
   <h1 class="govuk-heading-l">
     <span class="govuk-caption-l">{{verb}} task</span>
-    Check your changes
+    Check your answers
   </h1>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
 
       <div id="reassign-confirm-hint" class="govuk-hint">
-        Check you are reassigning work to the right person.
+        Check you are {{verb | lowercase }}ing work to the right person.
```

Comment:
Not a big fan of 'calculating' words - plays havoc when it comes round to translating phrases

Reply:
@paulhowes-HMCTS - hmm... good point. Happy to look at an alternative approach if you have one.

Reply:
Maybe just stick the verb logic in the code file and choose the relevant phrase as opposed to just the word.

Reply:
@paulhowes-HMCTS - I've reimplemented this as suggested; please re-review - thanks.